flaky assert "assert failed, TS used in import should in [now-1d, now]"

## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

<!-- a step by step guide for reproducing the bug. -->
in real-tikv test, the clock might out of sync between tidb and pd, so this check fails
```
[2024/12/20 10:33:01.029 +00:00] [ERROR] [wait_group_wrapper.go:246] ["panic in error group"] [recover="assert failed, TS used in import should in [now-1d, now], but got 454738780250439970"]


454738780250439970 is 2024-12-20 18:33:01.595000
```

### 2. What did you expect to see? (Required)

### 3. What did you see instead (Required)

### 4. What is your TiDB version? (Required)
master
<!-- Paste the output of SELECT tidb_version() -->

--- a/pkg/lightning/backend/local/region_job.go
+++ b/pkg/lightning/backend/local/region_job.go
@@ -440,14 +440,14 @@ func (local *Backend) doWrite(ctx context.Context, j *regionJob) error {
 	intest.AssertFunc(func() bool {
 		timeOfTS := oracle.GetTimeFromTS(dataCommitTS)
 		now := time.Now()
-		if timeOfTS.After(now) {
+		if timeOfTS.Sub(now) > time.Hour {
 			return false
 		}
 		if now.Sub(timeOfTS) > 24*time.Hour {
 			return false
 		}
 		return true
-	}, "TS used in import should in [now-1d, now], but got %d", dataCommitTS)
+	}, "TS used in import should in [now-1d, now+1h], but got %d", dataCommitTS)
 	if dataCommitTS == 0 {
 		return errors.New("data commitTS is 0")
 	}

